app/models/payment.rb

```diff
@@ -0,0 +1,3 @@
+class Payment < ApplicationRecord
+  has_many :bookings, dependent: :destroy
```

Comment:
payment là như kiểu phương thức thanh toán: tiền mặt, epay,  ... thôi đúng ko em? chư chưa phải là thông tin thanh toán giao dịch đúng ko?

Reply:
Dạ đúng rồi chị